map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp

```diff
@@ -41,6 +41,7 @@
 #include <rclcpp/rclcpp.hpp>
 
 #include <lanelet2_core/LaneletMap.h>
+#include <lanelet2_core/geometry/LineString.h>
```

Comment:
I see this kind of broken map on RViz with your lanelet map even when I use this PR. Do you know why?
![image](https://github.com/autowarefoundation/autoware.universe/assets/44218668/b375cbea-1c19-4761-af6d-dd100505a905)

(The map seems correct from the VMB though)

Reply:
Never mind, I forgot the set the projection mode to `local` in the config file.